runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsPlugin.java

```diff
@@ -116,10 +116,17 @@ ObjectIndicatorsConfig provideConfig(ConfigManager configManager)
 		return configManager.getConfig(ObjectIndicatorsConfig.class);
 	}
 
+	private static boolean reloadMapPointsOnStartup = false;
```

Comment:
From what I can tell, this can be removed in favor of checking `if (points.isEmpty())` in `reloadMapPoints()`. In fact, renaming that method to `loadMapPoints()` and not clearing the points map inside of it would be best.